lib/timeval.c

```diff
@@ -48,7 +48,7 @@ struct timeval curlx_tvnow(void)
 
```

Comment:
Changes I needed to get it working in Visual Studio are at [bagder/timeval-curlval...jay:pr_1693_amended](https://github.com/curl/curl/compare/bagder/timeval-curlval...jay:pr_1693_amended?expand=1)